src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1174,6 +1174,13 @@ PUBLIC_API STATUS freeTransceiver(PRtcRtpTransceiver*);
  */
 PUBLIC_API STATUS initKvsWebRtc(VOID);
 
+/*
+ * deinitKvsWebRtc deinitializes global state needed for all RtcPeerConnection's it must only be called once
+ *
+ * @return - STATUS code of the execution
+ */
+PUBLIC_API STATUS deinitKvsWebRtc(VOID);
```

Comment:
Let's come up with a better name for this API. So far we have pairs of  createXXX freeXXX which create and free the entire object. The initialization has been named like initializeXXX and releaseXXX. Though, I don't see a major difference - just suggest to have full names - like initialize instead of init